src/Microsoft.AspNetCore.Authentication.Abstractions/ChallengeBehavior.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Authentication
+{
+    public enum ChallengeBehavior
```

Comment:
Yeah but we likely can't do this until there's a bunch of corresponding AuthZ changes, we should tackle that in a follow on PR specifically for AuthZ (see https://github.com/aspnet/Security/issues/901)